### PR TITLE
Revert change for GetLastUpdated

### PR DESCRIPTION
**Pull request by bikamani**
We do not want to change behavior of this API hence reverting change and added xml comments.

### CODE REVIEW

**Review by drwill-ms — APPROVED** [review]

File: shared/src/TwinCollection.cs [hunk 1]
```diff
@@ -176,11 +177,11 @@ public Metadata GetMetadata()
         /// Gets the LastUpdated time for this property
         /// </summary>
         /// <returns>DateTime instance representing the LastUpdated time for this property</returns>
-        /// <exception cref="System.ArgumentNullException">Thrown when the TwinCollection metadata is null. 
+        /// <exception cref="System.NullReferenceException">Thrown when the TwinCollection metadata is null.
```
Comment (bikamani) [inline shared/src/TwinCollection.cs:180]: Reverting this change so no more behavior changes. It will still throw nre as before, we are documenting it.

**Pull request merged**